data/human/hails.txt

```diff
@@ -949,7 +949,12 @@ phrase "friendly civilian"
 		"How many New Bostonians does it take to change a lightbulb? I have no idea, never been there."
 		"How many Delvians does it take to change a lightbulb? Zero, the room caves in before you can replace the bulb."
 		"How many Sinterians does it take to change a lightbulb? Zero, they just send in a robot to do the job."
+		"How many Lunatics does it take to change a lightbulb? Don't bother, they light up with anger if you call them Lunatics."
+		"How many New Britons does it take to change a lightbulb? Many, they prefer selling instead of installing lightbulbs on their planet of shopkeepers."
+		`Someone met a djinn and said "leave me alone". That person is now stranded on an uninhabited planet.`
```

Comment:
```suggestion
		`Someone met a djinn and said "leave me alone." That person is now stranded on an uninhabited planet.`
```
Not sure about this, but based on the line above, which is definitely correct...